src/index.html

```diff
@@ -16,7 +19,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="body">
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="seconds-hand"></div>
+      <div class="minutes-hand"></div>
+      <div class="center"></div>
+    </div>
```

Comment:
You should consider using more semantic tags for your HTML structure. Although `div` is not incorrect, using more descriptive tags such as `section` or `article` would make your code more readable and accessible. For example, the `stopwatch` could be a `section`.